index.html

```diff
@@ -0,0 +1,278 @@
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./assets/css/styles.css" />
+    <title>Cherelle Simpson: Portfolio</title>
+  </head>
+
+  <body>
+    <!-- nav containing photo, name and 3 links to content -->
+    <header class="header">
+      <nav class="navbar-container">
+        <h1 id="name">Cherelle Simpson</h1>
+        <div class="nav-link-container">
+          <div class="nav-links">
+            <a class="nav-link" href="#aboutme-title">About Me</a>
+          </div>
+          <div class="nav-links">
+            <a class="nav-link" href="#portfolio-title">Portfolio</a>
+          </div>
+          <div class="nav-links">
+            <a class="nav-link" href="#contact-title">Contact</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <!-- hero background image -->
+    <div id="hero" title="Kinkakuji. Kyoto, Japan">
+      <figcaption id="herocaption">
+        Photo taken on my travels in Kyoto, Japan<br />&copy; Cherelle Simpson
+      </figcaption>
+    </div>
+
+    <main>
+      <!-- first section containing 'about me' -->
+      <section class="aboutme-container">
+        <h2 class="section-title" id="aboutme-title">About Me</h2>
+        <div class="aboutme-flex-container">
+          <img
+            id="imgCS"
+            src="./assets/images/CS.jpg"
+            alt="Photo of Cherelle Simpson"
+          />
+          <div class="aboutme">
+            <div class="aboutme-content">
+              <p>
+                An established business professional with extensive experience
+                in management roles within large and reputable companies.
+                Naturally creative and an adept problem-solver with proficiency
+                in a wide range of coding languages, including:
+              </p>
+              <ul>
+                <li><strong>HTML</strong> including semantic refactoring</li>
+                <li>
+                  <strong>CSS</strong> including flexbox - media queries -
+                  variables
+                </li>
+                <li>
+                  <strong>JavaScript</strong> including loops - arrays -
+                  conditional statements
+                </li>
+              </ul>
+              <p>
+                You can find a copy of my CV
+                <a
+                  href="assets/files/cvCherelleSimpson.pdf"
+                  id="cv"
+                  target="_blank"
+                  >here</a
+                >.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <!-- second section containing app image, title, tech and link -->
+      <section class="portfolio-container">
+        <h2 class="section-title" id="portfolio-title">Portfolio</h2>
+        <div class="portfolio-flex-container">
+          <!-- main image -->
+          <div class="portfolio-main-container">
```

Comment:
I would rename this class name as currently it isn't super clear what it is. We usually use the word container to describe something that contains other elements (as you have above with the flex container). This one and the one you have as `portfolio-sub-container` would be better named as `portfolio-card` or something